client/components/section-header/index.js

```diff
@@ -0,0 +1,66 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import EllipsisMenu from 'components/ellipsis-menu';
+import { H } from 'components/section';
+import { validateComponent } from 'lib/proptype-validator';
+
+/**
+ * A basic card component with a header. The header can contain a title, an action, and an `EllipsisMenu` menu.
```

Comment:
This needs to be updated, this is copied from `Card` currently.

Reply:
Updated